internal/gateway/proxy/direct.go

```diff
@@ -61,30 +137,59 @@ func (cp *ControllerProxy) UpdateEventbus(
 	return nil, errMethodNotImplemented
 }
 
+func authListSegment(_ context.Context, req interface{}) (authorization.ResourceKind, vanus.ID, authorization.Action) {
+	id := vanus.NewIDFromUint64((req.(*ctrlpb.ListSegmentRequest)).GetEventbusId())
+	return authorization.ResourceEventbus, id, authorization.EventbusGet
+}
+
 func (cp *ControllerProxy) ListSegment(
 	ctx context.Context, req *ctrlpb.ListSegmentRequest,
 ) (*ctrlpb.ListSegmentResponse, error) {
 	return cp.eventlogCtrl.ListSegment(ctx, req)
 }
 
+func authCreateSubscription(_ context.Context, req interface{},
+) (authorization.ResourceKind, vanus.ID, authorization.Action) {
+	id := vanus.NewIDFromUint64((req.(*ctrlpb.CreateSubscriptionRequest)).GetSubscription().GetNamespaceId())
+	return authorization.ResourceNamespace, id, authorization.SubscriptionCreate
```

Comment:
ResourceSubscription

Reply:
the id is namespace ID, the action SubscriptionCreate belong to resource namespace